assets/example_fastq_screen_references.csv

```diff
@@ -0,0 +1,4 @@
+name,dir,basename,aligner
+Ecoli,s3://ngi-igenomes/igenomes/Escherichia_coli_K_12_MG1655/NCBI/2001-10-15/Sequence/Bowtie2Index/,genome,bowtie2
```

Comment:
But we download testdata already from https://github.com/nf-core/test-datasets, right? Is it s3 buckets specifically you want to avoid? This is an open dataset, so running `aws s3` with `--no-sign-request` should be fine.

We could do either or both of the following:
1. Update the docs to show how to download the AWS data and tweak the .csv to run the test profile locally
2. Add the FastQ Screen references to our branch on https://github.com/nf-core/test-datasets

Any thoughts?

Reply:
Yes, basically all I need is that `nextflow run . -profile test,docker --outdir <OUTDIR>` work straight out of the box. When I first ran this command, I got the aws error messages I mentioned above. Now we just tried it on @KarNair 's computer and it just worked, so it is probably just a problem on my computer... I've now deleted my `~/.aws/credentials` and it seems to work alright. So there's probably nothing to fix, sorry for the trouble :sweat_smile: 

My point was that running the test profile should be very straightforward, to avoid scaring away new users, but if this error I stumbled upon only occurs for me it's probably fine :)

Reply:
That's a valid concern and I agree!

I'll be happy to tweak it if it proves to be an issue in the future. Merged for now, thanks!